### ISSUE TITLE
Create a scene that loads the 487 vox models from mmmm converted to schematics

### ISSUE THREAD

**Issue by acs** [Voxelers/mcthings#114]
The idea is to work directly in mcthings_scenes. The scene should download all the vox models and load them inside Minecraft.

Not sure if it is a good idea to convert the 487 models to schematic before (with vox2schematic) and load the schematic files. Maybe we can follow both approaches.

**Comment by acs** [comment]
Let's explore the distribution of creations:

* vox files
* schematic files
* Inside a Minecraft world: Java and Bedrock ideally

**Comment by acs** [comment]
Ok, mostly working. I need to adjust the positioning to avoid overlaps but the results are promising!

![Screenshot from 2020-07-07 08-04-23](https://user-images.githubusercontent.com/209533/86723849-90df4e80-c028-11ea-8bae-69feaa531c7e.png)

**Comment by acs** [comment]
Ok, much better now (I have removed the needed flip, but it has a bug updating the end_position):

![Screenshot from 2020-07-07 08-10-40](https://user-images.githubusercontent.com/209533/86724941-85405780-c029-11ea-9b49-7c1ce424fa04.png)
![Screenshot from 2020-07-07 08-09-55](https://user-images.githubusercontent.com/209533/86724957-88d3de80-c029-11ea-86a7-d41ad62af10a.png)
![Screenshot from 2020-07-07 08-09-21](https://user-images.githubusercontent.com/209533/86724968-8b363880-c029-11ea-9b39-464bcb77da53.png)
![Screenshot from 2020-07-07 08-08-55](https://user-images.githubusercontent.com/209533/86724975-8cfffc00-c029-11ea-80b2-c35303f90cf1.png)

**Comment by acs** [comment]
![Screenshot from 2020-07-07 08-24-13](https://user-images.githubusercontent.com/209533/86730255-9cce0f00-c02e-11ea-92c2-b683dea6703f.png)
![Screenshot from 2020-07-07 08-23-16](https://user-images.githubusercontent.com/209533/86730273-a0fa2c80-c02e-11ea-90c6-4f0cab654508.png)
![Screenshot from 2020-07-07 08-22-26](https://user-images.githubusercontent.com/209533/86730283-a2c3f000-c02e-11ea-8c5a-4a49f0faee53.png)
![Screenshot from 2020-07-07 08-21-42](https://user-images.githubusercontent.com/209533/86730291-a48db380-c02e-11ea-8024-cf21e874b7b8.png)

**Comment by acs** [comment]
Now in rows, easier to travel in them: just 48 rows!

![Screenshot from 2020-07-07 19-47-41](https://user-images.githubusercontent.com/209533/86821720-0419ac80-c08b-11ea-8cbc-6f199b7feb86.png)

**Comment by acs** [comment]
The final step is to create the empty world with all models loaded!

**Comment by acs** [comment]
Trying to create a world with all the models:

```
2020-07-08 06:09:25,722 Number of models: 487 
2020-07-08 06:09:25,723 Creating model alien_bot1.vox
2020-07-08 06:09:25,751 Creating model alien_bot2.vox
2020-07-08 06:09:25,773 Creating model alien_bot3.vox
....
2020-07-08 06:23:32,206 Creating model veh_car1.vox
2020-07-08 06:23:32,415 Creating model veh_car2.vox

Process finished with exit code 143 (interrupted by signal 15: SIGTERM)
```

so it seems not all of them were added.

```
[minecraft-launcher]$ grep "Creating model" /tmp/m | wc -l
455
```

Probably we need to add more memory to Minecraft. But no traces.

But taking a look to the results, the models are correctly loaded. Not sure if there are empty models at some points. 

For example, we have a hole in this row.

![Screenshot from 2020-07-08 06-33-26](https://user-images.githubusercontent.com/209533/86876401-f133c600-c0e4-11ea-810f-fa927321d08a.png)

In any case, as a demo, the results are good enough. Some points to improve:

* Add more space between rows

**Comment by acs** [comment]
![Screenshot from 2020-07-08 06-35-48](https://user-images.githubusercontent.com/209533/86876611-67382d00-c0e5-11ea-8700-df604fef80be.png)

The last row:

![Screenshot from 2020-07-08 06-39-11](https://user-images.githubusercontent.com/209533/86876807-c8600080-c0e5-11ea-919b-7120366dfa2e.png)

**Comment by acs** [comment]
Ok, let's save the world.

```
[saves]$ du -sh vox/
74M	vox/
[saves]$ zip -r vox.zip vox/
[saves]$ du -sh vox.zip 
5.3M	vox.zip
[saves]$ mv vox.zip vox-world.zip
```

Great. It is created using 1.12.2 so it is easily imported in newer version. And it can also be converted to Bedrock.

**Comment by acs** [comment]
Next step is to deploy it in a server to test it.

**Comment by acs** [comment]
World available in github (1.12.2 version): https://github.com/Voxelers/mcthings_scenes/blob/develop/worlds/vox-world.zip

Scene created at: https://github.com/Voxelers/mcthings_scenes/blob/develop/apps/scene_magicavoxel.py

Tweet: https://twitter.com/McthingsP/status/1280902177972682752
